tutorials/polkadot-sdk/system-chains/asset-hub/send-a-tx-paying-fee-different-token.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Send a Transaction While Paying the Fee with a Different Token
+description: This tutorial demonstrates how to send a DOT transfer transaction while paying the fees using a different token on the Asset Hub.
+---
+
+# Send a Transaction While Paying the Fee with a Different Token
+
+## Introduction
+
+The Asset Hub provides a powerful feature that allows users to pay transaction fees using alternative tokens instead of the native token of the chain.
+
+This tutorial demonstrates how to send a DOT transfer transaction while paying the fees using a different token (USDT in this example) on the Asset Hub.
+
+## Environment Setup
+
+Let's set up the development environment for this tutorial:
+
+1. Create a new directory and initialize the project:
+
+    ```bash
+    mkdir fee-payment-tutorial && \
+    cd fee-payment-tutorial
+    ```
+
+2. Initialize the project:
+
+    ```bash
+    npm init -y
+    ```
+
+3. Install dev dependencies:
+
+    ```bash
+    npm install --save-dev @types/node@^22.12.0 ts-node@^10.9.2 typescript@^5.7.3
+    ```
+
+4. Install dependencies:
+
+    ```bash
+    npm install --save @polkadot-labs/hdkd@^0.0.13 @polkadot-labs/hdkd-helpers@^0.0.13 polkadot-api@1.9.5
+    ```
+
+5. Create TypeScript configuration:
+
+    ```bash
+    npx tsc --init && npm pkg set type=module
+    ```
+
+    The command `npm pkg set type=module` is used to set the type of the project to module. This is necessary to use the `import` statement in the TypeScript code.
+
+6. Generate Polkadot API types for Asset Hub:
+
+    ```bash
+    npx papi add assetHub -n polkadot_asset_hub
+    ```
+
+7. Create a new file called `fee-payment-transaction.ts`:
+
+    ```bash
+    touch fee-payment-transaction.ts
+    ```
+
+## Local Asset Hub Setup
+
+Before running the script, you'll need to fork the Asset Hub locally using Chopsticks:
+
+```bash
+chopsticks -c polkadot-asset-hub
+```
+
+This command forks the Asset Hub chain, making it available at `ws://localhost:8000`. By running `polkadot-asset-hub`, you're using the Asset Hub fork with the configuration specified in the [`polkadot-asset-hub.yml`](https://github.com/AcalaNetwork/chopsticks/blob/master/configs/polkadot-asset-hub.yml){target=_blank} file.  This configuration defines Alice's account with USDT assets. If you want to use a different chain, ensure the account you're using has the necessary assets.
+
+## Implementation
```

Comment:
Implementation of what? Like what is the user achieving here? I think that's what the title should reflect